server/routes/createGoal/createGoalController.ts

```diff
@@ -54,6 +54,36 @@ export default class CreateGoalController {
     return res.render('pages/goal/create/index', { ...view.renderArgs })
   }
 
+  // TODO: RR-734 - Create controller and view classes new create goal journey
+  newGetCreateGoalView: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber } = req.params
+    const { prisonerSummary } = req.session
+
+    req.session.newGoals = req.session.newGoals || []
+
+    if (!req.session.newGoal?.createGoalForm) {
+      req.session.newGoal = {
+        createGoalForm: { prisonNumber },
+      } as NewGoal
+    }
+
+    const today = moment().toDate()
+    const futureGoalTargetDates = [
+      futureGoalTargetDateCalculator(today, 3),
+      futureGoalTargetDateCalculator(today, 6),
+      futureGoalTargetDateCalculator(today, 12),
+    ]
+
+    const view = new CreateGoalView(
```

Comment:
I think we might want a different view model class here (not sure yet? 🤔 ; but I'm wondering if the changes we are about to get into might conflict with what we already do for the current create goal (singular) journey, and therefore this view class)

Can you put a TODO on here please, to work out whether we need a new view class, or can re-use this existing one